app/src/common/shared/com/igalia/wolvic/utils/DeviceType.java

```diff
@@ -30,57 +34,57 @@ public class DeviceType {
     public static final int MetaQuest3 = 18;
 
     private static @Type int mType = Unknown;
+    private static String mDeviceName = "Unknown Type";
```

Comment:
Nit: "Unknown device" (I know it is wrong in the current code)

Reply:
Done.